src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -109,8 +109,8 @@
 
   <!-- Common dependencies across all targets -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.*" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.*" />
+    <ProjectReference Include="C:\Users\t-jarroyo\OneDrive - Microsoft\Julio_Internship\durabletask\src\DurableTask.Core\DurableTask.Core.csproj" />
+    <ProjectReference Include="C:\Users\t-jarroyo\OneDrive - Microsoft\Julio_Internship\durabletask\src\DurableTask.AzureStorage\DurableTask.AzureStorage.csproj" />
```

Comment:
we'll eventually want to remove these references to your local drive :-) . But since this is still a draft PR, I suppose it's not high priority